### ISSUE TITLE
Different availability for different provider

### ISSUE THREAD

**Issue by Amaroart** [Crocoblock/suggestions#983]
Hello and thanks for the good work!
I have copied/pasted a comment made by someone on the youtube video: https://www.youtube.com/watch?v=ykMb0LFcMIo&t=618s

i have the thought about the plugin.
i hope you will be able to upgrade it soon.

 - There is no way to input a Providers Availability. The plugin assumes all providers are available from start of use. For Example: If One Chef can only do Wedding Service on the Weekends. I feel there should be a way to input a Providers Availability.

- There is no way to add additional costs. Using your Use-case as an Example. Say we choose Birthday Party as the Service, however what if we want to make one Provider a different cost. For Example you are booking a Birthday Party and you can choose Standard Chef or Professional Chef, the Professional Chef should cost more.

- How do you pass the form fields to the woocommerce checkout form? I saw in your example 'phone number' was not passed through, but 'email' was. I cannot see any documentation on this.

best regards .
Amaro.

**Comment by oceandiveloper** [comment]
yes please add custom availability for providers. It should be possible to block off-days, add specific work hours. In a perfect world you would also be able to block slots based on events you already have in a Google calendar.

**Comment by rodolfofloresbravo** [comment]
This is a must

**Comment by MakhonkoDenis** [comment]
Hi guys. 
- In version 1.2.2 the custom schedule for providers and services was added.
- You can add a custom price for any post type. https://prnt.sc/vy9pi0 After that, if you are using a single page form, you can add hidden fields and get the price value from the post meta data. https://prnt.sc/vy9vl3
- It is necessary to enable the integration with WooCommerce in the JetAppointment settings. After that you need to open the form and set up the notification "Insert appointment" https://prnt.sc/vy9zrv

**Comment by jpfrancois2021** [comment]
Can each provider edit their own schedule, custom price etc. through a frontend form? It is obvious that many site owners will prefer not to give admin access to providers, hence having the possibility to do this on frontend will be a great addition @MakhonkoDenis

**Comment by Ahmad-Hisham007** [comment]
> Can each provider edit their own schedule, custom price etc. through a frontend form? It is obvious that many site owners will prefer not to give admin access to providers, hence having the possibility to do this on frontend will be a great addition [@MakhonkoDenis](https://github.com/MakhonkoDenis)

This will be great. If this is possible. Is it? +1
